terminal_client.go

```diff
@@ -119,7 +119,7 @@ func (c *TerminalClient) listCmd() error {
 
 	tbl := TableDef{
 		Columns: []ColumnDef{
-			ColumnDef{"id", 25},
+			ColumnDef{"id", 42},
```

Comment:
Instead of that many extra chars, why not just mark the active one with a preceding asterisk. e.g.

```
  some-deploy-tag
* another-deploy-tag
  yet-another-deploy

```